tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsEventBridgeTests.cs

```diff
@@ -0,0 +1,93 @@
+// <copyright file="AwsEventBridgeTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AWS
+{
+    [Trait("RequiresDockerDependency", "true")]
+    [UsesVerify]
+    public class AwsEventBridgeTests : TracingIntegrationTest
+    {
+        public AwsEventBridgeTests(ITestOutputHelper output)
+            : base("AWS.EventBridge", output)
+        {
+        }
+
+        public static IEnumerable<object[]> GetEnabledConfig()
+            => from packageVersionArray in PackageVersions.AwsEventBridge
+               from metadataSchemaVersion in new[] { "v0", "v1" }
+               select new[] { packageVersionArray[0], metadataSchemaVersion };
+
+        public override Result ValidateIntegrationSpan(MockSpan span, string metadataSchemaVersion) => span.Tags["span.kind"] switch
+        {
+            SpanKinds.Consumer => span.IsAwsEventBridgeInbound(metadataSchemaVersion),
+            SpanKinds.Producer => span.IsAwsEventBridgeOutbound(metadataSchemaVersion),
+            SpanKinds.Client => span.IsAwsEventBridgeRequest(metadataSchemaVersion),
+            _ => throw new ArgumentException($"span.Tags[\"span.kind\"] is not a supported value for the AWS EventBridge integration: {span.Tags["span.kind"]}", nameof(span)),
+        };
+
+        [SkippableTheory]
+        [MemberData(nameof(GetEnabledConfig))]
+        [Trait("Category", "EndToEnd")]
+        public async Task SubmitsTraces(string packageVersion, string metadataSchemaVersion)
+        {
+            SetEnvironmentVariable("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", metadataSchemaVersion);
+            var isExternalSpan = metadataSchemaVersion == "v0";
+            var clientSpanServiceName = isExternalSpan ? $"{EnvironmentHelper.FullSampleName}-aws-eventbridge" : EnvironmentHelper.FullSampleName;
+
+            using var telemetry = this.ConfigureTelemetry();
+            using (var agent = EnvironmentHelper.GetMockAgent())
+            using (await RunSampleAndWaitForExit(agent, packageVersion: packageVersion))
+            {
+#if NETFRAMEWORK
+                var expectedCount = 8;
+                var frameworkName = "NetFramework";
```

Comment:
I'm assuming you are on a Mac so you can't run/generate snapshots for .NET Framework
If they don't get generated/created on the CI build I can run and generate them locally and add them

Reply:
Yes, I am on Mac. Can you add those missing snapshots? Thank you!

Reply:
Added!